lms/resources/_js_config/__init__.py

```diff
@@ -284,7 +284,7 @@ def add_deep_linking_api(self):
         self._config.setdefault("filePicker", {})
         self._config["filePicker"]["deepLinkingAPI"] = config
 
-    def enable_instructor_toolbar(self, enable_editing=False, enable_grading=False):
+    def enable_instructor_toolbar(self, enable_editing=True, enable_grading=False):
```

Comment:
I reckon there some value to keep this parameter here even we happen to always use it with `True` now.